### PR TITLE
Forward design review

### PR DESCRIPTION
**Pull request by mikeger**
Fixes for displaying different forwarded content.

### CODE REVIEW

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4821,10 +4823,12 @@
 				TargetAttributes = {
 					1EE85C4F1B5F96FB008E084B = {
 						CreatedOnToolsVersion = 6.4;
+						DevelopmentTeam = W5KEQBF9B5;
```
Comment (typfel) [inline Wire-iOS.xcodeproj/project.pbxproj]: Should not be commited?

**Comment by typfel** [comment]
Did you run the snapshot tests?

**Comment by mikeger** [comment]
@typfel yes

**Comment by mikeger** [comment]
@typfel removed Xcode DevelopmentTeam

**Pull request merged**